test/unit/gateways/adyen_test.rb

```diff
@@ -376,6 +376,16 @@ def test_splits_sent
     end.respond_with(successful_authorize_response)
   end
 
+  def test_execute_threed_false_with_additional_data
```

Comment:
If I revert the changes in the Adyen gateway here, this test still passes. Is there any way to catch a failure in the old code?

Reply:
Good catch, I'll take a look

Reply:
Strange, I'm getting inconsistent behavior between virtually identical unit and remote tests. Even as-is here, the unit test doesn't add the executeThreeD flag - (`assert_false` is a bad test).

Reply:
Nevermind, it was just a typo on my part `execute_threeds` instead of `execute_threed` 😩

Reply:
Fixed now. If you could verify I'd appreciate it!

Reply:
Worked for me - thanks for fixing this 🎉